Makefile

```diff
@@ -50,12 +50,15 @@ tdd:
 test_failed:
 	$(DOCKER_PREFIX) py.test --lf $(ARGS)
 
+HIGHEST_MIGRATION = $$(cd migrations; \
+		       ls | sort -rn | sed -n 's/^\([0-9]\+\).*/\1/p' | head -1)
```

Comment:
My first thought was that I'd update schematic, but then I looked at schematic and this seemed like the cleaner option. But I guess I'll give it a try.